lib/evm/interpreter.js

```diff
@@ -107,6 +101,20 @@ module.exports = class Interpreter {
     return getPrecompile(address.toString('hex'))
   }
 
+  runPrecompile (code, data, gasLimit) {
+    if (typeof code !== 'function') {
+      throw new Error('Invalid precompile')
+    }
+
+    const opts = {
+      data,
+      gasLimit,
+      _common: this._vm._common
+    }
+
+    return code(opts)
+  }
+
```

Comment:
Much more fitting to have this here, always felt this being far too high out within the execution stack.